docker/addons/cassandra-writer/subjects.toml

```diff
@@ -0,0 +1,4 @@
+# If you want to listen on all subjects, just pass one element ["*"], otherwise
+# pass the list of subjects.
```

Comment:
I think it would be a good to leave here (in a comment) an example of this list (maybe with and without subtopics), so that we have a documentation how this `filter` field can be specified.

Reply:
Good point. I have planned to edit the section filter of the writers in the documentation. I didn't know if it is better to explain here or in the documentation.

Reply:
Personally, I think it better to add documentation inside the `docs` repository.

Reply:
This should be done as well. But I prefer that this config file holds one comment as well about the format that it accepts.

Reply:
Maybe something like that can be enough:
`If you want to listen on all subjects, just pass one element [">"], otherwise pass the list of subjects (e.g ["*.bedroom.temperature" "*.livingroom.*"]).` ?

Reply:
@jonathandreyer please use version with full subject in the config file.